WordPress/Classes/Utility/Media/AsyncImageView.swift

```diff
@@ -5,23 +5,59 @@ import Gifu
 /// (see ``AnimatedImage``).
 @MainActor
 final class AsyncImageView: UIView {
-    let imageView = GIFImageView()
-
+    private let imageView = GIFImageView()
```

Comment:
I enhanced the API so that the implementation details (`GIFImageView`) don't leak and `AsyncImageView` has full control over the image. The `image` property will now also correctly support `AnimagedImage`, both setter and getter.